kgforge/specializations/models/demo_model.py

```diff
@@ -81,7 +81,7 @@ def mapping(self, entity: str, source: str, type: Callable) -> Mapping:
 
     # Validation.
 
-    def _validate_one(self, resource: Resource) -> None:
+    def _validate_one(self, resource: Resource, **kwargs) -> None:
```

Comment:
Why `**kwargs` here and not a type argument here as in rdf_model ?

I think we should have a parallel between demo_model and rdf_model w.r.t this `_validate_one` method.

Reply:
I resolved it in the last push